src/Components/Onboarding/ItemLink.tsx

```diff
@@ -78,7 +78,15 @@ export default class ItemLink extends React.Component<Props, State> {
         fadeOut={this.state.fadeOut}
         item_id={this.props.id}
       >
-        <Col>{<Avatar src={this.props.image_url ? this.props.image_url : ""} width={50} height={50} />}</Col>
+        <Col>
+          {
+            <Avatar
+              src={this.props.image_url ? this.props.image_url : "https://www.artsy.net/images/icon-70.png"}
```

Comment:
Adding an external link for now, but the plan is to create a lower level component that display a default image when the `image_url` isn't available.